apps/ledger-live-desktop/scripts/post-install.js

```diff
@@ -24,7 +24,7 @@ const rebuildDeps = async (folder, file) => {
 };
 
 async function main() {
-  const folder = "node_modules/.cache/";
+  const folder = ".turbo/cache/";
```

Comment:
what is this change for?

Reply:
It's a breaking change from the upgrade: https://turbo.build/blog/turbo-2-0#breaking-changes

> Turborepo’s default disk cache location moved to .turbo/cache from node_modules/.cache